internal/mq/msgstream/msgstream.go

```diff
@@ -77,4 +77,5 @@ type Factory interface {
 	NewMsgStream(ctx context.Context) (MsgStream, error)
 	NewTtMsgStream(ctx context.Context) (MsgStream, error)
 	NewQueryMsgStream(ctx context.Context) (MsgStream, error)
+	NewMsgStreamDisposer(ctx context.Context) func([]string, string) error
```

Comment:
Why not add a Disposer to mqstream interface? the factory interface should use to msgstream of different type , unsubscribe should be a mq action